### PR TITLE
implement preliminary functions to call cpe and cve api from NIST

### PR DESCRIPTION
**Pull request by JeffC25**

### CODE REVIEW

File: vulnerability_checker/src/app.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import requests
+import os
+import logging
+from dotenv import load_dotenv
+
```
Comment (ujalil101) [inline vulnerability_checker/src/app.py:5]: I recommend putting a little description, saying what this module is doing

**Review by ujalil101 — CHANGES_REQUESTED** [review]
Besides the comment i put, everything looks good!

File: vulnerability_checker/src/app.py (hunk 1 above)
Comment (JeffC25) [reply vulnerability_checker/src/app.py:5]: I added some comments to the code.

**Review by ujalil101 — APPROVED** [review]
Much better! I think its all good to merge!

**Pull request merged**